.cz-config.js

```diff
@@ -0,0 +1,66 @@
+module.exports = {
+  // add additional standard scopes here
+  scopes: [{name: 'accounts'}, {name: 'admin'}],
+  // use this to permanently skip any questions by listing the message key as a string
+  skipQuestions: [],
+
+  /* DEFAULT CONFIG */
+  messages: {
+    type: 'What type of changes are you committing:',
+    scope: '\nEnlighten us with the scope (optional):',
+    customScope: 'Add the scope of your liking:',
+    subject: 'Write a short and simple description of the change:\n',
+    body: 'Provide a LONGER description of the change (optional). Use "|" to break new line:\n',
+    breaking: 'List any BREAKING CHANGES (optional):\n',
+    footer: 'List any ISSUES CLOSED by this change (optional). E.g.: #31, #34:\n',
+    confirmCommit: 'Are you sure you the above looks right?',
+  },
+  types: [
+    {
+      value: 'fix',
+      name: '🐛   fix:        Changes that fix a bug',
+      emoji: '🐛',
+    },
+    {
+      value: 'feat',
+      name: ' 🚀  feat:       Changes that introduce a new feature',
+      emoji: '🚀',
+    },
+    {
+      value: 'refactor',
+      name: '🤷‍♂️   refactor:   Changes that fix a bug nor adds a feature',
+      emoji: '🤷‍♂️',
+    },
+    {
+      value: 'test',
+      name: '🧪   test:       Adding missing tests',
+      emoji: '🧪',
+    },
+    {
+      value: 'style',
+      name:
+        '💅   style:      Changes that do not impact the code base  \n                   (white-space, formatting, missing semi-colons, etc)',
+      emoji: '💅',
+    },
+    {
+      value: 'docs',
+      name: '📝   docs:       Changes to the docs',
+      emoji: '📝',
+    },
+    {
+      value: 'chore',
+      name:
+        '🤖   chore:      Changes to the build process or auxiliary tools\n                   and or libraries such as auto doc generation',
+      emoji: '🤖',
+    },
+  ],
+  allowTicketNumber: false,
+  isTicketNumberRequired: false,
+  ticketNumberPrefix: '#',
+  ticketNumberRegExp: '\\d{1,5}',
+  allowCustomScopes: true,
+  allowBreakingChanges: ['feat', 'fix', 'chore'],
+  breakingPrefix: '🚧 BREAKING CHANGES 🚧',
+  footerPrefix: 'CLOSES ISSUE:',
+  subjectLimit: 77,
```

Comment:
seems a bit short, not sure what benefit limiting this is supposed to give. I also think it gets confusing if both husky and commitizen enforce their own rules

Reply:
It's basically a default commitlint [rule](https://commitlint.js.org/#/reference-rules). Altough they keep it at 72. 

What would have your preference? 100?

Reply:
Is it possible to have no limit? What advantages does having a limit give us?

Reply:
I think you might be confusing the naming of the setting. It limits the amount of characters on the first commit line. so `feat(): MAX CHARS`

In the conventional commits config it's limited to 100. So that's what you're used to.

Reply:
aaah ok yeah then I think 100 is good?

Reply:
https://github.com/storyofams/next-boilerplate/pull/6/commits/48012e3cd62afc55d42f4ebf21f041fe0de330cb

Reply:
https://github.com/storyofams/next-boilerplate/pull/6/commits/48012e3cd62afc55d42f4ebf21f041fe0de330cb